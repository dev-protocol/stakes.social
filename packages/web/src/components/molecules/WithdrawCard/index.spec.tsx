import React from 'react'
import { render } from '@testing-library/react'
import { WithdrawCard } from './'
import BigNumber from 'bignumber.js'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${WithdrawCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(<WithdrawCard label="Holder" onSubmitWithdraw={() => {}} amount={new BigNumber(1000)} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
