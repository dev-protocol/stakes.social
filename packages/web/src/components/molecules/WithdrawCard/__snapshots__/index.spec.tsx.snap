// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithdrawCard Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card ant-card-bordered"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-row"
        >
          <div
            class="ant-col"
            style="flex: 1 1 252px;"
          >
            <span
              class="sc-bdvvtL fRYnpk"
            >
              Withdraw 
              Holder
               Reward
            </span>
            <div
              class="sc-gsDKAQ bHBuhs"
            >
              1000
               DEV
            </div>
          </div>
          <div
            class="ant-col"
            style="flex: 1 1 252px;"
          >
            <div
              class="ant-space ant-space-vertical"
            >
              <div
                class="ant-space-item"
              >
                <button
                  class="ant-btn ant-btn-primary ant-btn-lg"
                  type="button"
                >
                  <span>
                    Withdraw
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
